aspnetcore/razor-components/components.md

```diff
@@ -46,6 +46,25 @@ Component members can then be used as part of the component's rendering logic us
 
 After the component is initially rendered, the component regenerates its render tree in response to events. Razor Components then compares the new render tree against the previous one and applies any modifications to the browser's Document Object Model (DOM).
 
+## Integrate components into Razor Pages and MVC apps
+
+Use components with existing Razor Pages and MVC apps. There's no need to rewrite existing pages or views to use Razor Components. When the page or view is rendered, components are prerendered&dagger; at the same time. 
+
+> [!NOTE]
+> &dagger;Server-side prerendering is enabled for Razor Components apps by default. Client-side Blazor apps will support prerendering in the upcoming Preview 4 release. For more information, see [Update templates/middleware to use MapFallbackToPage/File](https://github.com/aspnet/AspNetCore/issues/8852).
+
+To render a component from a page or view, use the `RenderComponentAsync<TComponent>` HTML helper method:
+
+```cshtml
+<div id="Counter">
+    @(await Html.RenderComponentAsync<Counter>(new { IncrementAmount = 10 }))
+</div>
+```
+
+Components rendered from pages and views aren't yet interactive in the Preview 3 release. For example, selecting a button doesn't trigger a method call. A future preview will address this limitation along with adding support for rendering components using the normal element and attribute syntax.
```

Comment:
Components rendered from pages and views aren't yet interactive in the Preview 3 release. For example, selecting a button doesn't trigger a method call. A future preview will address this limitation and add support for rendering components using the normal element and attribute syntax.

Reply:
along is mostly banned